pkg/controller/network_policy.go

```diff
@@ -202,6 +202,41 @@ func (c *Controller) handleUpdateNp(key string) error {
 		return err
 	}
 
+	// set svc addreess_set
+	svcAsName := strings.Replace(fmt.Sprintf("%s.%s.service", np.Name, np.Namespace), "-", ".", -1)
+	svcIps, err := c.fetchSelectedSvc(np.Namespace, &np.Spec.PodSelector)
+	if err != nil {
+		klog.Errorf("failed to fetchSelectedSvc svcIps result  %v", err)
+		return err
+	}
+	if err := c.ovnClient.CreateAddressSet(svcAsName, np.Namespace, np.Name, "service"); err != nil {
+		klog.Errorf("failed to create address_set %s, %v", svcAsName, err)
+		return err
+	}
+	klog.Infof("fetchSelectedSvc -> svcIps is %v", svcIps)
+	if err := c.ovnClient.SetAddressesToAddressSet(svcIps, svcAsName); err != nil {
+		klog.Errorf("failed to set netpol svc, %v", err)
+		return err
+	}
```

Comment:
We should split service IPs and service address set into two part: IPv4 and IPv6. Cluster IPs can be retrieved from service's`spec.clusterIPs`.

Reply:
But i find pod IPs and service address not be set into two part,so it's  necessary?

Reply:
We use port group to implement Pod selector of network policy. Ingress and egress rules are splited into IPv4 and IPv6 by function `fetchPolicySelectedAddresses`.

Reply:
ok